src/tests/conformance/cmd/transaction.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{
+    protocol::{
+        codecs::binary::{BinaryMessage, Payload},
+        proto::TransactionStatus::TsCurrent,
+    },
+    setup::{constants::TESTNET_READY_TIMEOUT, testnet::TestNet},
+    tools::{
+        rpc::{submit_transaction, wait_for_account_data},
+        synth_node::SyntheticNode,
+    },
+};
+
+#[tokio::test]
+#[allow(non_snake_case)]
+async fn c019_MT_TRANSACTION_node_should_broadcast_transaction_to_all_peers() {
+    // ZG-CONFORMANCE-019
+
+    const FIRST_NODE_ID: usize = 0;
+    const SECOND_NODE_ID: usize = 1;
+    const TRANSACTION_BLOB: &str = "12000022000000002400000001201B0000001E61400000012A05F20068400000000000000A73210330E7FC9D56BB25D6893BA3F317AE5BCF33B3291BD63DB32654A313222F7FD020744630440220297389244D36AF12115296F409C446D9A5D808880DC7FF323AA207ED529CE6C802207AAC5D2A96CB102CBDE85D2A4BA814253CA133AC9277041CAE2E1A349FB233FF8114B5F762798A53D543A014CAF8B297CFF8F2F937E883149193D6AED0CBBC25790ADE05D020C9C6D9201DCF";
+
+    // Start a testnet.
+    let mut testnet = TestNet::new().unwrap();
+    testnet.start().await.unwrap();
+    wait_for_account_data(
+        &testnet.running[FIRST_NODE_ID].rpc_url(),
+        "rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh",
```

Comment:
where does this come from?

Reply:
This is a genesis account. In most tests it has '//TODO make constant', but not here.